OpenRA.Game/Network/ReplayRecorder.cs

```diff
@@ -69,6 +69,9 @@ void StartSavingReplay(byte[] initialContent)
 
 		public void Receive(int clientID, byte[] data)
 		{
+			if (disposed)
+				return;
```

Comment:
Sorry for not getting to that yet - that's pretty lame of me.

Could you add this comment:  `TODO: This can be removed once NetworkConnection is fixed to dispose properly.`

That'll remind me to get to it eventually, and we can take your PR in the meantime.